components/eventing-controller/api/v1alpha2/subscription_types.go

```diff
@@ -88,6 +89,18 @@ func (s Subscription) MarshalJSON() ([]byte, error) {
 	return json.Marshal(a)
 }
 
+// GetUniqueTypes returns the de-duplicated types from subscription spec.
+func (s *Subscription) GetUniqueTypes() []string {
```

Comment:
I think I created a similar function! but I put it in the nats utils at the moment

Reply:
But I think it should be here